src/Windows/MainWindow.Helpers.cs

```diff
@@ -54,12 +55,15 @@ private static void AdjustManualSize(MsBox.Avalonia.Dto.MessageBoxStandardParams
         {
             if (sizeToContent == SizeToContent.Manual)
             {
-                int screenWidth = PInvoke.GetSystemMetrics(global::Windows.Win32.UI.WindowsAndMessaging.SYSTEM_METRICS_INDEX.SM_CXVIRTUALSCREEN),
-                    screenHeight = PInvoke.GetSystemMetrics(global::Windows.Win32.UI.WindowsAndMessaging.SYSTEM_METRICS_INDEX.SM_CYVIRTUALSCREEN);
-                msgboxparams.MinWidth = 300;
-                msgboxparams.MinHeight = 200;
-                msgboxparams.Width = screenWidth / 2;
-                msgboxparams.Height = screenHeight / 2;
+            int screenWidth = 1920, screenHeight = 1080;
```

Comment:
Currently, no cross-platform way exists to get the screen dimensions that I am aware of.
This implements a fallback to the most common resolution used (via Steam HW Survey), but will use actual size on Windows systems.